### PR TITLE
redirect to consent page if user not in Redcap

### PR DESCRIPTION
**Pull request by SimonLab**
see #35

and [this comment](https://github.com/research-resource/research_resource/issues/35#issuecomment-289410450)

### CODE REVIEW

File: web/controllers/account_controller.ex [hunk 1]
```diff
@@ -7,20 +7,26 @@ defmodule ResearchResource.AccountController do
...
       conn.assigns.current_user.ttrrid
       |> @redcap_api.get_user_data()
-      # Turn user data from a map to a keyword list
-      |> Enum.to_list
-      |> Enum.map(fn {key, val} -> {String.to_atom(key), val} end)
 
-    render conn, "index.html", user_details: user_details
+    cond do
+      user_data ->
+        user_details =
```
Comment (samhstn) [inline web/controllers/account_controller.ex:16]: could we remove this assignment?

**Review by samhstn — COMMENTED** [review]
Looks good 👍

File: web/controllers/account_controller.ex (hunk 1 above)
Comment (SimonLab) [reply web/controllers/account_controller.ex:16]: I use user_details a bit later in the render function, do you think of a better way to pass the value to the view?

Comment (samhstn) [reply web/controllers/account_controller.ex:16]: Sorry, my mistake didn't see this, looks good 👍

**Review by Danwhy — APPROVED** [review]
:+1:

**Pull request merged**